app/basicComponents/Input.js

```diff
@@ -31,6 +31,42 @@ const ActualInput = styled.input`
   outline: none;
 `;
 
+const RadioInput = styled.input`
```

Comment:
better extract this as new basic component - RadioButton

Reply:
fix